hoprd-node-1.yaml

```diff
@@ -14,20 +14,70 @@ spec:
     enabled: true
   monitoring:
     enabled: true
----
-apiVersion: hoprnet.org/v1alpha
-kind: Hoprd
-metadata:
-  name: hoprd-node-not-announced
-  namespace: master-staging
-spec:
-  network: master-staging
-  version: providence
-  config:
-    announce: false
-  enabled: true 
-  ingress:
-    enabled: true
-  monitoring:
-    enabled: true
+  deployment:
+    resources: |
+      limits:
+        cpu: 2000m
+        memory: 2Gi
+      requests:
+        cpu: 750m
+        memory: 256Mi
+    startupProbe: |
+      exec:
+        command:
+          - sh
+          - '-c'
+          - >-
+            curl -s http://localhost:3001/api/v2/node/info -H
+            "X-Auth-Token: $HOPRD_API_TOKEN" | jq -r .connectivityStatus |
+            grep -q Green
+      initialDelaySeconds: 5
+      timeoutSeconds: 5
+      periodSeconds: 30
+      successThreshold: 1
+      failureThreshold: 60
+    livenessProbe: |
+      exec:
+        command:
+          - sh
+          - '-c'
+          - >-
+            curl -s http://localhost:3001/api/v2/node/info -H
+            "X-Auth-Token: $HOPRD_API_TOKEN" | jq -r .connectivityStatus |
+            grep -q Green
+      initialDelaySeconds: 5
+      timeoutSeconds: 5
+      periodSeconds: 10
+      successThreshold: 1
+      failureThreshold: 5
+    readinessProbe: |
+      exec:
+        command:
+          - sh
+          - '-c'
+          - >-
+            curl -s http://localhost:3001/api/v2/node/info -H
+            "X-Auth-Token: $HOPRD_API_TOKEN" | jq -r .connectivityStatus |
+            grep -q Green
+      initialDelaySeconds: 5
+      timeoutSeconds: 5
+      periodSeconds: 10
+      successThreshold: 1
+      failureThreshold: 5
+# ---
+# apiVersion: hoprnet.org/v1alpha
+# kind: Hoprd
+# metadata:
+#   name: hoprd-node-not-announced
+#   namespace: master-staging
+# spec:
+#   network: master-staging
+#   version: providence
+#   config:
+#     announce: false
+#   enabled: true 
+#   ingress:
+#     enabled: true
+#   monitoring:
+#     enabled: true
```

Comment:
Intentional?

Reply:
I didn't want to loose that configuration. I have this file for testing purposes, every time that I need to develop or test a new feature it has different values. And at this moment I don't need to test the announce/not-announce